imod/tests/test_mf6/test_import_prj.py

```diff
@@ -0,0 +1,170 @@
+import importlib
+from zipfile import ZipFile
+
+import numpy as np
+import pooch
+from numpy.testing import assert_allclose
+
+from imod.formats.prj import open_projectfile_data
+
+REGISTRY = pooch.create(
+    path=pooch.os_cache("imod"),
+    base_url="https://github.com/Deltares/imod-artifacts/raw/main/",
+    version=None,
+    version_dev="main",
+    env="IMOD_DATA_DIR",
+)
+
+with importlib.resources.files("imod.data") as pkg_dir:
+    REGISTRY.load_registry(pkg_dir / "registry.txt")
+fname_model = REGISTRY.fetch("iMOD5_model.zip")
```

Comment:
This code is duplicate with the code in imod.data. I think there is more opportunity for code sharing.

Reply:
done